<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoDialog</class>
 <widget class="QDialog" name="TodoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todo</string>
  </property>
  <property name="windowIcon">
   <iconset theme="view-task" resource="../breeze-qownnotes.qrc">
    <normaloff>:/icons/breeze-qownnotes/16x16/view-task.svg</normaloff>:/icons/breeze-qownnotes/16x16/view-task.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QFrame" name="editFrame">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="priorityLabel">
            <property name="text">
             <string>Priority</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="prioritySlider">
            <property name="toolTip">
             <string>Priority</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="accessibleName">
             <string extracomment="Priority"/>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>9</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="sliderPosition">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="reminderLabel">
            <property name="text">
             <string>Reminder</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFrame" name="reminderFrame">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>12</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="reminderCheckBox">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateTimeEdit" name="reminderDateTimeEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QOwnNotesMarkdownTextEdit" name="descriptionEdit">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="html">
           <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="summaryEdit">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>summary</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QFrame" name="horizontalFrame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="saveButton">
             <property name="toolTip">
              <string>save current todo item</string>
             </property>
             <property name="text">
              <string>&amp;save</string>
             </property>
             <property name="icon">
              <iconset theme="document-save" resource="../breeze-qownnotes.qrc">
               <normaloff>:/icons/breeze-qownnotes/16x16/document-save.svg</normaloff>:/icons/breeze-qownnotes/16x16/document-save.svg</iconset>
             </property>
             <property name="shortcut">
              <string notr="true">Ctrl+S</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveAndInsertButton">
             <property name="toolTip">
              <string>save current todo item</string>
             </property>
             <property name="text">
              <string>save and &amp;insert into note</string>
             </property>
             <property name="icon">
              <iconset theme="document-save" resource="../breeze-qownnotes.qrc">
               <normaloff>:/icons/breeze-qownnotes/16x16/document-save.svg</normaloff>:/icons/breeze-qownnotes/16x16/document-save.svg</iconset>
             </property>
             <property name="shortcut">
              <string notr="true">Ctrl+I</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeButton">
             <property name="toolTip">
              <string>remove current todo item</string>
             </property>
             <property name="text">
              <string>&amp;remove</string>
             </property>
             <property name="icon">
              <iconset theme="edit-delete" resource="../breeze-qownnotes.qrc">
               <normaloff>:/icons/breeze-qownnotes/16x16/edit-delete.svg</normaloff>:/icons/breeze-qownnotes/16x16/edit-delete.svg</iconset>
             </property>
             <property name="shortcut">
              <string notr="true">Ctrl+R</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QFrame" name="descriptionEditSearchFrame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="selectFrame">
       <layout class="QGridLayout" name="selectLayout">
        <item row="2" column="0">
         <widget class="QListWidget" name="todoList"/>
        </item>
        <item row="5" column="0">
         <widget class="QFrame" name="frame">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="showCompletedItemsCheckBox">
             <property name="text">
              <string>Show completed items</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="reloadTodoListButton">
             <property name="toolTip">
              <string>reload the todo list from server</string>
             </property>
             <property name="text">
              <string>&amp;reload</string>
             </property>
             <property name="icon">
              <iconset theme="view-refresh" resource="../breeze-qownnotes.qrc">
               <normaloff>:/icons/breeze-qownnotes/16x16/view-refresh.svg</normaloff>:/icons/breeze-qownnotes/16x16/view-refresh.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="todoListSelector">
          <property name="toolTip">
           <string>select your todo list</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QProgressBar" name="todoItemLoadingProgressBar">
          <property name="toolTip">
           <string>Todo list items are being loaded from the server</string>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="newItemEdit">
          <property name="placeholderText">
           <string>search or create todo item</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QOwnNotesMarkdownTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">widgets/qownnotesmarkdowntextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>newItemEdit</tabstop>
  <tabstop>todoList</tabstop>
  <tabstop>summaryEdit</tabstop>
  <tabstop>prioritySlider</tabstop>
  <tabstop>reminderDateTimeEdit</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>reloadTodoListButton</tabstop>
  <tabstop>showCompletedItemsCheckBox</tabstop>
  <tabstop>todoListSelector</tabstop>
  <tabstop>reminderCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="../breeze-qownnotes.qrc"/>
 </resources>
 <connections/>
</ui>
